packages/dds/tree/src/core/schema-stored/schema.ts

```diff
@@ -119,6 +126,8 @@ export interface SchemaPolicy {
 
 	/**
 	 * If true, new content inserted into the tree should be validated against the stored schema.
+	 * @remarks
+	 * TODO: This is not information used to interpret the stored schema: this configuration should be moved elsewhere.
```

Comment:
We should create an actual item for this and link it (as well as for allowUnknownOptionalFields)